solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/Cell/RepositoryViewCell.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  RepositoryViewCell.swift
+//  GitHubApp
+//
+//  Created by Felipe on 06/07/22.
+//
+
+import UIKit
+
+struct RepositoryViewCellConfiguration{
+    let repositoryName: String
+    let repsitoryOwnerName: String
+}
+
+
+class RepositoryViewCell: UITableViewCell {
```

Comment:
Como sua classe nós garantimos que não vai ter herança, considere colocar final 

```suggestion
final class RepositoryViewCell: UITableViewCell {
```